README.md

```diff
@@ -1,3 +1,6 @@
-# cilium-perf-test
-
-Cilium performance evaluation repo.
+<p align="center">
+  <h1 align="center">Cilium Performance</h1>
+  <p align="center">
+    <p align="center">A repo dedicated to Cilium performance testing and evaluation.</p>
+  </p>
+</p>
```

Comment:
While it looks prettier, this is a markdown file and we should keep the format as markdown, which is much easier to edit.

The content itself looks good, though